### ISSUE TITLE
ValidationError on `lessLoaderOptions.javascriptEnabled`

### ISSUE THREAD

**Issue by daniel-hauser** [DocSpring/craco-antd#33]
2 days ago [craco-less](https://github.com/DocSpring/craco-less/commit/f72885c0a5bd1c20ba1e4d69fa61a171dc9f1d3b) upgraded [less-loader from 5 to 6](https://github.com/webpack-contrib/less-loader/releases/tag/v6.0.0) that has a braking change.

This causes the following error:
```
ValidationError: Invalid options object. Less Loader has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'javascriptEnabled'. These properties are valid:
   object { lessOptions?, prependData?, appendData?, sourceMap?, implementation? }
```

There are 2 fixes that can be done:
1. Pin the version of `craco-less` to 1.16.0
2. Use the new `lessOptions` as described [here](https://github.com/webpack-contrib/less-loader#lessoptions)

**Comment by ndbroadbent** [comment]
Hello, yes it can be difficult to keep everything in sync. In the meantime, please use an older version of craco-antd, or I would be happy to merge a pull request that adds support for the new version of craco-less. Thanks!

**Comment by daniel-hauser** [comment]
Thanks for the quick answer.
For now - ill open a PR that pins the craco-less version, ill be happy if you approve it

**Comment by ndbroadbent** [comment]
Hello, I've updated craco-less and the less loader to the latest versions, and the changes have been pushed to `craco-antd` version `1.15.0`

**Comment by adijesori** [comment]
@ndbroadbent I'm having the same error mentioned by @daniel-hauser also in the new version `1.15.0`

```
ValidationError: Invalid options object. Less Loader has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'javascriptEnabled'. These properties are valid:
   object { lessOptions?, prependData?, appendData?, sourceMap?, implementation? }
```

**Comment by daniel-hauser** [comment]
@ndbroadbent the issue wasn't fixed, can you please re-open it?

As I mentioned earlier, there are only 2 options:
1. pin `craco-less` to 1.16.0 (uses `less-loader@5`) - a hotfix, done in #34 
2. change the code [here](https://github.com/DocSpring/craco-antd/blob/ea24b8de95367202f0b429d8e67a9a1faa93a804/lib/craco-antd.js#L53:L70) to work with `less-loader@6` (see changes [here](https://github.com/webpack-contrib/less-loader#lessoptions)) - the real fix, but can brake code for current users

**Comment by adijesori** [comment]
@ndbroadbent Thanks for reopening! Can #34 or #35 be merged please? I'm not sure why they were closed without merging

**Comment by davidvazinwnwd** [comment]
I have the same issue too!! what can I do?>

**Comment by aitsvet** [comment]
Why pin to older version, let's correct this:
https://github.com/DocSpring/craco-antd/pull/37

**Comment by ndbroadbent** [comment]
Hi everyone, sorry about that! I thought this was fixed but I guess the tests didn't catch the problems so these need to be improved. I've merged #36 to pin craco-less to < 1.17.0. #37 was a good attempt (Thanks @aitsvet!), but unfortunately it needs some changes to preserve any existing lessOptions: https://github.com/DocSpring/craco-antd/pull/37#pullrequestreview-421724640
